sensor/common/scannerdefinitions/http_handler.go

```diff
@@ -0,0 +1,91 @@
+package scannerdefinitions
+
+import (
+	"io"
+	"net/http"
+	"net/url"
+	"time"
+
+	"github.com/pkg/errors"
+	"github.com/stackrox/rox/pkg/httputil"
+	"github.com/stackrox/rox/pkg/set"
+	"github.com/stackrox/rox/pkg/utils"
+	mtls2 "github.com/stackrox/scanner/pkg/mtls"
+	"google.golang.org/grpc/codes"
+)
+
+// scannerDefinitionsHandler handle requests to retrieve scanner vulnerability
+// definitions from Central.
+type scannerDefinitionsHandler struct {
+	http.Handler
+	centralClient *http.Client
+	centralHost   string
+}
+
+// NewDefinitionsHandler creates a new scanner definitions handler.
+func NewDefinitionsHandler(centralHost string) (http.Handler, error) {
+	// Set up an HTTP client for Central.
+	clientConfig, err := mtls2.TLSClientConfigForCentral()
+	if err != nil {
+		return nil, errors.Wrap(err, "generating TLS client config for Central")
+	}
+	client := &http.Client{
```

Comment:
We do something different for the kocache: https://github.com/stackrox/stackrox/blob/3.69.1/sensor/common/sensor/sensor.go#L109

Should we just copy that instead?